lib/lograge.rb

```diff
@@ -116,7 +116,7 @@ def self.unsubscribe(component, subscriber)
 
   def self.setup(app)
     self.application = app
-    app.config.action_dispatch.rack_cache[:verbose] = false if app.config.action_dispatch.rack_cache
+    app.config.action_dispatch.rack_cache[:verbose] = false if rack_cache_hashlike?(app)
```

Comment:
That would not work - because then `:[]=` would always be called on `rack_cache`, raising the error that this commit is trying to fix.